src/model/channel/message.rs

```diff
@@ -72,6 +72,12 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    /// Please note:
+    /// Not all channel mentions in a message will appear in mention_channels.
+    /// Only textual channels that are visible to everyone in a lurkable guild will ever be included.
+    /// Only crossposted messages (via Channel Following) currently include mention_channels at all.
+    /// If no mentions in the message meet these requirements, this field will not be sent.
+    /// [`See more in the discord documentation`]: https://discord.com/developers/docs/resources/channel#message-object
```

Comment:
[Reference-style links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) need to be on their own line to render properly.

Regular links are like ``[`See more in the discord documentation`](https://discord.com/developers/docs/resources/channel#message-object)`` with both the text and url together, but reference-style links need to have their url defined separate from the text.

```suggestion
    /// [`See more in the discord documentation`].
    ///
    /// [`See more in the discord documentation`]: https://discord.com/developers/docs/resources/channel#message-object
```